lib/src/fnc/util/http/mod.rs

```diff
@@ -14,7 +14,7 @@ pub(crate) fn uri_is_valid(uri: &str) -> bool {
 fn encode_body(req: RequestBuilder, body: Value) -> RequestBuilder {
 	match body {
 		Value::Bytes(bytes) => req.header(CONTENT_TYPE, "application/octet-stream").body(bytes.0),
-		_ if body.is_some() => req.json(&body),
+		_ if body.is_some() => req.json(&serde_json::Value::from(body)),
```

Comment:
We have just merged an `into_json()` method on `main`. Kindly change this to

```rust
_ if body.is_some() => req.json(&body.into_json()),
```

Reply:
I've just pushed this @rushmorem !